x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -287,12 +316,38 @@ function PageHeader({ handleGuidedSetupClick, onTimeRangeRefresh }: PageHeaderPr
         />
       </EuiFlexItem>
       <EuiFlexItem grow={false} style={{ alignItems: 'flex-end' }}>
-        <EuiButton color="text" iconType="wrench" onClick={handleGuidedSetupClick}>
-          <FormattedMessage
-            id="xpack.observability.overview.guidedSetupButton"
-            defaultMessage="Guided setup"
-          />
-        </EuiButton>
+        {showTour ? (
+          <EuiTourStep
```

Comment:
What if we use the [`anchor`](https://elastic.github.io/eui/#/display/tour) option to attach the tour to the button? If I understand correctly, if we use it, we wouldn't need to memoize the button.

Reply:
makes sense 👍